doc/Core.xml

```diff
@@ -512,12 +512,14 @@
     <para role="reference">&lt;<link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://www.ietf.org/rfc/rfc3986.txt"></link>&gt;</para>
     <para>IETF RFC  4122, A Universally Unique IDentifier (UUID) URN Namespace</para>
     <para role="reference">&lt;<link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://www.ietf.org/rfc/rfc4122.txt"></link>&gt;</para>
-    <para>IETF  4702, The Dynamic Host Configuration Protocol (DHCP) Client Fully Qualified Domain Name (FQDN) Option</para>
+    <para>IETF RFC 4702, The Dynamic Host Configuration Protocol (DHCP) Client Fully Qualified Domain Name (FQDN) Option</para>
     <para role="reference">&lt;<link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://www.ietf.org/rfc/rfc4702.txt"></link>&gt;</para>
     <para>IETF  4861, Neighbor Discovery for IP version 6 (IPv6)</para>
     <para role="reference">&lt;<link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://www.ietf.org/rfc/rfc4861.txt"></link>&gt;</para>
     <para>IETF  4862, IPv6 Stateless Address Auto configuration</para>
     <para role="reference">&lt;<link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://www.ietf.org/rfc/rfc4862.txt"></link>&gt;</para>
+    <para>IETF 7519, JSON Web Token (JWT)</para>
```

Comment:
change IETF 7519->IETF RFC 7519

Reply:
I see inconsistency everywhere. We use IETF XXX, RFC XXX, IETF RRC XXX and RFCXXXX.

I'd rather rely on RFC XXXX and align this usage across the whole document